go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rudderlabs/sqlconnect-go
 
 go 1.23.5
 
+replace github.com/gocql/gocql => github.com/scylladb/gocql v1.14.4
```

Comment:
what is this for?

Reply:
It's complaining for JetBrains IDE with the following error:` go: github.com/gocql/gocql@v1.14.2: invalid version: unknown revision v1.14.2`

We have added for the same in [rudder-server](https://github.com/rudderlabs/rudder-server/blob/1fca839521bcb0d28f606052f76fde3e9fd8dc6c/go.mod#L13) and [rudder-go-kit](https://github.com/rudderlabs/rudder-go-kit/blob/1e951d70080bbd98de2a42411f45d3857c623c83/go.mod#L5).